style.css

```diff
@@ -16,6 +16,95 @@ body {
   margin: 0;
 }
 
+/* popup style */
+
+.close {
```

Comment:
- [ ] The close button is broken, kindly try to fix it so your design looks more complete

![Screenshot_20220513_071631](https://user-images.githubusercontent.com/25227022/168222501-51db6311-eb10-44cc-a6d3-433d248c79c1.png)